index.coffee

```diff
@@ -64,9 +70,9 @@ module.exports = (message, options = {}) ->
 
   # flush: log message when stream ends
   logCount = (cb) ->
-    if counter > 0
+    if ((counter > 0 and not options.includeZero) or (options.includeZero and not options.onlyZero) or (counter is 0 and options.includeZero and options.onlyZero))
```

Comment:
this is a terrifyingly long condition... i suspect it can be simplified somewhat via boolean arithmetic. another option is to pull individual conditions out to variables so it's more legible:

`if hasFiles or hasZero or hasOnlyZero`